notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>@leavingseason, what I proposed was the following:</p><p><code></code><code></p><p>from reco_utils.recommender.deeprec.models.sequential.sli_rec import SLI_RECModel</p><p>from reco_utils.recommender.deeprec.models.sequential.asvd import ASVDModel</p><p>from reco_utils.recommender.deeprec.models.sequential.caser import CaserModel</p><p>from reco_utils.recommender.deeprec.models.sequential.gru4rec import GRU4RecModel</p><p></code><code></code></p><p> </p><p>then in the cell below where you are instantiating the model, we add the 4 models but comment 3, so the user can have them by just uncommenting them:</p><p> </p><p><code></code><code></p><p>model = SLI_RECModel(hparams, input_creator, seed=RANDOM_SEED)</p><p>#model = ASVDModel(hparams, input_creator, seed=RANDOM_SEED)</p><p>#model = CaserModel(hparams, input_creator, seed=RANDOM_SEED)</p><p>#model = GRU4RecModel(hparams, input_creator, seed=RANDOM_SEED)</p><p> </p><p> </p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I see.